app/src/main/res/values/strings.xml

```diff
@@ -665,6 +666,8 @@
 	<string name="error_account_name_mismatch">The existing account for %s doesn\'t match the one requested for %s.</string>
 	<string name="error_account_set_auth_token_security_exception">Uh-oh! This isn\'t an error we expect to see. If you have ScorePal installed, there\'s a known problem that one prevents the other from signing in. We\'re working to resolve the issue.</string>
 	<string name="error_field_required">This field is required</string>
+	<!-- Buttons -->
+	<string name="button_settings">Settings</string>
```

Comment:
Yeah OK. I'm not a big fan of reusing strings when they're used for a different purpose just because they have the same text, but it makes sense in this case. I'll use the existing one.